src/Libraries/DSIronPython/DSIronPython_DynamoCustomization.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0"?>
-<doc>
-    <assembly>
-        <name>DSIronPython</name>
-    </assembly>
-    <namespaces>
-        <namespace name="DSIronPython">
-            <category>Core.Scripting</category>
```

Comment:
@reddyashish why is this being deleted?

Reply:
Not sure where this xml is used but the python node is being saved in the Script -> Editor folder in the library section(same as before).